CNAME

```diff
@@ -1 +0,0 @@
-viperdev.io
```

Comment:
Okay! actually did that because I've seen that when github kind of tries to build jekyll site under that CNAME, and if two repos have same CNAME it crashes.
But I'll try here, because this is not our gh-pages branch, so this might not try to build itself